app/classifier/tasks/survey/choice.jsx

```diff
@@ -73,6 +72,28 @@ class Choice extends React.Component {
     this.props.onConfirm(this.props.choiceID, this.state.answers);
   }
 
+  handleRadioKeyDown(e) {
+    switch (e.which) {
+      case BACKSPACE:
+      case SPACE:
+        if (e.target.checked) {
+          e.preventDefault();
+          this.resetSingleAnswerQuestion(e);
+        }
+        break;
+      default:
+    }
+  }
+
+  resetSingleAnswerQuestion(e) {
+    const { type, name, value } = e.target;
+    const { answers } = this.state;
+    if (type === 'radio' && answers[name] === value) {
```

Comment:
I think this might need a second conditional where
```
else if (type === 'radio' && answers.name.includes(value)) {
  const newValues = answers.name.filter(item => item !== value);
  answers.name = newValues;
  this.setState({ answers });
}
```

or something similar to handle a question that supports multiple answers like the second question (What behaviors do you see?) in this test workflow:
https://survey-radio-toggle.pfe-preview.zooniverse.org/projects/markb-panoptes/survey-task-test-project/classify

Reply:
I can toggle the checkbox answers without any extra code. There's already code to splice the answers array in `handleAnswers`, I think.

Reply:
nope the code noted above isn't fixing it, dang, I'll keep trying a bit

Reply:
Verified on your test workflow that I can toggle the checkboxes on and off, and the final classification reflects the checked choices.

Reply:
NB Radio buttons will only allow you to select one answer anyway, so `answers[name]` will never be an array for `type=radio`.

Reply:
I don't think I can toggle answers for a question that accepts multiple answers with the spacebar on this branch, but can on master. Works fine with the mouse on both branches.

Reply:
Ah right, in that case the error is in the `keyDown` handler, not this function.

Reply:
I've added a comment to remind myself to fix this.